services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy

```diff
@@ -214,6 +215,29 @@ class SwitchSyncSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "System doesn't synchronize missing default rules"() {
```

Comment:
The presence of missing default rules will not affect anything. But the presence of excess default rules will lead to the fact that during synchronization, along with excess default rules, the proper default rules are deleted.